### PR TITLE
ODE-1234 Integrated the odevalidator with conditional and static config

### PR DESCRIPTION
**Pull request by hmusavi**
Note: Adding the validator as a submodule allows the harness to have access to the validator config.ini.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -13,3 +13,6 @@
 [submodule "jpo-sdw-depositor"]
 	path = jpo-sdw-depositor
 	url = https://github.com/usdot-jpo-ode/jpo-sdw-depositor.git
+[submodule "qa/test-harness/ode-output-validator-library"]
```
Comment (mvs5465) [inline .gitmodules]: Lets not include this as a submodule if possible. This is not very portable.

See https://github.com/usdot-jpo-ode/ode-output-validator-library/pull/15 for info about how to add the file to the library.

File: qa/test-harness/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,3 @@
-git+https://github.com/usdot-jpo-ode/ode-output-validator-library@master#egg=odevalidator
```
Comment (mvs5465) [inline qa/test-harness/requirements.txt]: Should be added back.

File: qa/test-harness/test-harness.py [hunk 3]
```diff
@@ -8,7 +8,8 @@
 from argparse import ArgumentParser
 from kafka import KafkaConsumer
 from pathlib import Path
-from odevalidator import TestCase
+from odevalidator.validator import TestCase
```
Comment (mvs5465) [inline qa/test-harness/test-harness.py]: Should be `from odevalidator import TestCase`

File: qa/test-harness/test-harness.py [hunk 4]
```diff
@@ -30,16 +31,21 @@ def listen_to_kafka_topic(topic, msg_queue):
...
     parser.add_argument("--output-file", dest="output_file_path", help="Output file to which detailed validation results will be printed.", metavar="LOGFILEPATH", required=False)
     args = parser.parse_args()
-    assert Path(args.config_file_path).is_file(), "Configuration file '%s' could not be found" % args.config_file_path
 
     # Parse test config and create test case
-    validator = TestCase(args.config_file_path)
+    if args.config_file_path:
+        validator = TestCase(args.config_file_path)
+    else:
+        validator = TestCase("ode-output-validator-library/odevalidator/config.ini")
```
Comment (mvs5465) [inline qa/test-harness/test-harness.py]: Will need to be replaced with correct path once added to the library.

File: .gitmodules (hunk 1 above)
Comment (hmusavi) [reply .gitmodules]: Will do at a later time.

File: qa/test-harness/test-harness.py (hunk 3 above)
Comment (hmusavi) [reply qa/test-harness/test-harness.py]: Relative import does not work for running odevalidator.validator tester. For now, non-relative imports seem to work everywhere. Will revisit relative imports at a later time.

**Pull request merged**